client/lib/js/utils/cryptoconditions/type_ids.js

```diff
@@ -0,0 +1,13 @@
+
+const TypeIds = {
```

Comment:
I wonder if it'd be useful to make some sort of JS driver/parser for the interledger spec.